internal/graph/check.go

```diff
@@ -375,6 +377,7 @@ func (c *LocalChecker) checkDirect(parentctx context.Context, req *ResolveCheckR
 		relation := tk.GetRelation()
 
 		fn1 := func(ctx context.Context) (*openfgapb.CheckResponse, error) {
+			span.SetAttributes(attribute.String("user_tuple", tk.String()))
```

Comment:
This attribute now belongs to the span named `checkDirectUserTuple`. Sso I think `user_tuple_ is a good attribute name, let me know if you think otherwise